conformance/tests/directives_type_ignore.py

```diff
@@ -0,0 +1,14 @@
+"""
+Tests "# type: ignore" comments.
+"""
+
+# Specification: https://typing.readthedocs.io/en/latest/spec/directives.html#type-ignore-comments
+
```

Comment:
Should we test `# type: ignore # <comment or other marker>` form since spec calls it out explicitly?

Reply:
Added missing test as suggested. Also, I noticed that I was missing a test for the file-level `# type: ignore` directive, so I added that as well.